src/components/NewCommentForm.tsx

```diff
@@ -1,13 +1,14 @@
 import classNames from 'classnames';
 import React, { useState } from 'react';
 import { CommentData } from '../types/Comment';
+import { useAppSelector } from '../app/hooks';
 
 type Props = {
   onSubmit: (data: CommentData) => Promise<void>;
 };
 
 export const NewCommentForm: React.FC<Props> = ({ onSubmit }) => {
-  const [submitting, setSubmitting] = useState(false);
+  const { isLoading } = useAppSelector(state => state.comments);
```

Comment:
It's better to create selectors https://redux-toolkit.js.org/api/createSelector instead of passing anonim function every time when you need something from the state

**Fix previous mentor's comment for all cases**